icu4c/source/common/unicode/uversion.h

```diff
@@ -63,25 +63,41 @@ typedef uint8_t UVersionInfo[U_MAX_VERSION_LENGTH];
 /*===========================================================================*/
 
 /**
- * \def U_NAMESPACE_BEGIN
- * This is used to begin a declaration of a public ICU C++ API.
+ * \def U_CXX_BEGIN
+ * This is used to begin a C++ block.
  * When not compiling for C++, it does nothing.
  * When compiling for C++, it begins an extern "C++" linkage block (to protect
  * against cases in which an external client includes ICU header files inside
  * an extern "C" linkage block).
  *
- * It also begins a versioned-ICU-namespace block.
+ * @stable ICU 2.7
+ */
+
+/**
+ * \def U_CXX_END
+ * This is used to end a declaration of a C++ API.
+ * When not compiling for C++, it does nothing.
+ * When compiling for C++, it ends the extern "C++" block begun by
+ * U_CXX_BEGIN.
+ *
+ * @stable ICU 2.7
```

Comment:
...And this would mean that `U_CXX_BEGIN` and `U_CXX_END` were introduced somewhen in the past (at latest in ICU 2.7) and have become part of stable API of ICU in ICU 2.7. (In translation to modern versioning, it would be ICU 27.)
However these are not even in ICU 64.

Reply:
Sorry, I did not know what to put there, please provide me with the text and I will fix.

Reply:
if it were accepted it would be `@draft`